src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -497,14 +501,32 @@ var DynamicTable = new Class({
     },
 
     selectRow: function(rowId) {
-        this.deselectAll();
         this.selectedRows.push(rowId);
+        this.setRowClass();
+        this.onSelectedRowChanged();
+    },
+
+    deselectRow: function(rowId) {
+        this.selectedRows.erase(rowId);
+        this.setRowClass();
+        this.onSelectedRowChanged();
+    },
+
+    selectRows: function(rowIdStart, rowIdEnd) {
+        this.deselectAll();
+
+        var select = false;
+        var that = this;
         this.tableBody.getElements('tr').each(function(tr) {
-            if (tr.rowId == rowId)
-                tr.addClass('selected');
-            else
-                tr.removeClass('selected');
+            if ((tr.rowId == rowIdStart) || (tr.rowId == rowIdEnd)) {
```

Comment:
I don't think this compare equal is good, what if `rowIdStart` or `rowIdEnd` doesn't match any `tr.rowId`?
I would suggest:
```js
// pseudo code
var rowIdStart = min(rowId1, rowId2)
var rowIdEnd = max(rowId1, rowId2)

this.tableBody.getElements('tr').each(function(tr) {
    if (rowIdStart <= tr.rowId && tr.rowId <= rowIdEnd) // add row
}
```

Reply:
> what if rowIdStart or rowIdEnd doesn't match any tr.rowId?

Then all remaining rows would be selected. 

> I would suggest:

In the torrents table the rowid is the hash of the torrent. So there won’t be any ordering to the rowids.